gm4_blast_furnaces/pack.mcmeta

```diff
@@ -0,0 +1,35 @@
+{
+    "pack": {
+        "pack_format": 0,
+        "description": "A Gamemode 4 Module"
+    },
+    "Blast Furnaces": "Blast Furnaces",
+    "blast_furnaces": "blast_furnaces",
+    "site_description": "IF I DON'T SET THIS AT SOME POINT I'M A MORON",
```

Comment:
Site description missing. "Double your ore output with this multi-block structure."

Reply:
Top notch placeholder from me there